oak_functions_enclave/.cargo/config.toml

```diff
@@ -2,4 +2,4 @@
 target = "x86_64-unknown-none"
 
 [target.x86_64-unknown-none]
-rustflags = "-C relocation-model=static -C target-feature=+sse,+sse2,+ssse3,+sse4.1,+sse4.2,+avx,+avx2,+rdrand,-soft-float"
+rustflags = "-C relocation-model=static -C code-model=small -C target-feature=+sse,+sse2,+ssse3,+sse4.1,+sse4.2,+avx,+avx2,+rdrand,-soft-float"
```

Comment:
What does code-model small do?

Reply:
See Section 3.5.1 in https://refspecs.linuxbase.org/elf/x86_64-abi-0.99.pdf for the gory details, but...

By default, `x86_64-unknown-none` defaults to the `kernel` code model, which is designed for, well, kernel code that lives in the last 2 GiB of the address space.

In contrast, the `small` code model expects all your symbols to be be lower than `0x7effffff`, which allows the compiler to generate more efficient code.